docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -0,0 +1,182 @@
+:::
+
+![Step 5](./img/webhooks/5.png "Event Subscription")
+  
+Finally, confirm <Hlt>Active</Hlt> is toggled and click <Hlt>Add webhook</Hlt>.
+
+![Step 6](./img/webhooks/6.png "Submit Form")
+
+3. Verifying Connectivity
+
+From your Webhooks Dashboard, click on the webhook you just created.
+
+![Step 7](./img/webhooks/7.png "Created")
+
+Navigate to <Hlt>Recent Deliveries</Hlt>.
+
+![Step 8](./img/webhooks/8.png "Recent Deliveries")
+
+Click on the <Hlt>ping</Hlt> event and ensure a successful response was returned.
+
+![Step 9](./img/webhooks/9.png "Response")
+
+
+### Apps
+
+It may be tedious to configure webhooks for each of your Github repositories. 
+You can instead opt to configure a 
+[Github App](https://docs.github.com/en/apps); allowing you to receive events 
+from all or select repositories.
+
+1. Navigate to the Github Apps Dashboard
+
+    1. Navigate to <Hlt>Settings</Hlt>. This will be listed in a dropdown menu that is toggled by clicking your Github avatar.
+    1. Click <Hlt>Developer Settings</Hlt>.This will be in the bottom left-hand corner of the settings dashboard.
+    1. Click <Hlt>Github Apps</Hlt>.
+
+![Step 1](./img/apps/123.png "Settings")
+
+2. Register a new Github App
+
+    1. Set a unique name in the <Hlt>Github App name</Hlt> field.
+    1. For the <Hlt>Homepage URL</Hlt> field you can use an arbitrary URL
+    or the repository URL.
+    ![Step 4](./img/apps/4.png "Register New App")
+    1. Set the <Hlt>Webhook URL</Hlt> to the value we retrieved from the 
+    [Retrieving the Receiver's URL](#retrieving-the-receivers-url) step.
+    1. Set the <Hlt>Secret</Hlt> field to the value we set to the `secret` key 
+    from the [Configuring the Receiver](#configuring-the-receiver) 
+    step.
+    ![Step 5](./img/apps/5.png "Configure Webhook")
+    1. Scroll down to <Hlt>Configure Permissions</Hlt> -> 
+    <Hlt>Repository Permissions</Hlt>
+     :::note
+      For the option to subscribe to repo push events, we will need 
+      <Hlt>read + write</Hlt> access for the <Hlt>Contents</Hlt> permission. For the option to subscribe to registry push events(ghcr), we will need <Hlt>read + write</Hlt>
+      access for the <Hlt>Packages</Hlt> permission.
+    :::
+    ![Step 7](./img/apps/7.png "Permissions - Contents")
+    ![Step 8](./img/apps/8.png "Permissions - Packages")
+    1. Scroll down to <Hlt>Subscribe to events</Hlt>
+    1. Select options based on your choices for the previous step.
+    :::note
+      If you set the <Hlt>Contents</Hlt> permission to <Hlt>read + write</Hlt> 
+      you will have the <Hlt>Push</Hlt> option available to you. If you set <Hlt>Packages</Hlt> to <Hlt>read + write</Hlt> you will have the <Hlt>Registry package</Hlt> option available to you.
+    :::
+    ![Step 9](./img/apps/9.png "Subscribe to Events")
+    1. Finally, click <Hlt>Create GitHub App</Hlt>.
+    ![Step 10](./img/apps/10.png "Submit Form")
```

Comment:
Both in terms of how it appears here in the source _and_ in how it renders, this all looks a bit cramped. Introducing additional line breaks between the items in the sub-lists will help combat the "wall of text" feeling.